prow/config/config.go

```diff
@@ -148,6 +152,11 @@ func (ownersDirBlacklist OwnersDirBlacklist) DirBlacklist(org, repo string) (bla
 	if bl, ok := ownersDirBlacklist.Repos[org+"/"+repo]; ok {
 		blacklist = append(blacklist, bl...)
 	}
+
+	preconfiguredDefaults := []string{"\\.git$", "_output$", "vendor/.*/.*"}
```

Comment:
The path will start with the org/repo name, so the regex shouldn't contain `^` in the beginning. PTAL.